app/services/doc_auth/lexis_nexis/doc_pii_reader.rb

```diff
@@ -20,7 +20,7 @@ module DocPiiReader
       private
 
       def read_pii(true_id_product)
```

Comment:
since it looks like sometimes it returns nil, should we docoument?
```suggestion
      # @return [Pii::StateId, nil]
      def read_pii(true_id_product)
```